app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -126,13 +126,20 @@ class MainActivity : AppCompatActivity() {
     }
 
     override fun onBackPressed() {
-        val currentFragment = this.supportFragmentManager.findFragmentById(R.id.frameContainer)
```

Comment:
why is this removed and then written again? Rest LGTM 👍

Reply:
First I declare root fragment then current fragment in sequence.

Reply:
Updated.